.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default ownership to "Product Direction"
+/content/en/contribute/medic @medic/product-direction
```

Comment:
Could we call the group `product-leadership`? It might be me, but I find the `direction` word confusing.

Reply:
I had that originally, but considered other options for a hierarchy with more or less people in the group. Given your feedback though, that sounds more confusing than helpful. I've updated this and the corresponding group in GitHub.